adventuregame.rb

```diff
@@ -1,73 +1,109 @@
+# frozen_string_literal: true
+
+# == Adventure \Game
+#
+# This is gonna be such a great game! Here's some details...
+# - TODO: add details
+#
+# == Running the game
+#
+# To run the game, ... (TODO: moar details needed; etc.)
+
 #GAME START
+
+# Player
+#
+# TODO: describe what this class is/does/is-responsible-for
+#
+# TODO: you might add an example of using this class here, too
```

Comment:
A class gets some documentation and explanation, too
- using ENGLISH(!)
- i.e. "The Player class will represent the state of the Player's character within the Game"

Here's an example of, :ahem:, an example...

```Ruby
# Example:
#   player_character = Player.new
#   player.hit_points = 5
#   player.hurt_self(4)
#   player.alive?       # => true
#   player.hit_points   # => 1
#   player.hurt_self(2)
#   player.alive?       # => false
```